plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -11,6 +11,9 @@ deploy.preferences.dialog.title.flexible=Deploy to App Engine Flexible
 deploy.preferences.dialog.title.withProject=Deployment Parameters for "{0}"
 deploy.preferences.dialog.label.selectAccount=Account:
 deploy.preferences.dialog.accountSelector.login=<Add a new account...>
+deploy.preferences.dialog.flex.pricing=There is no free quota for App Engine flexible \
```

Comment:
App Engine flexible environment deployments --> the App Engine flexible environment